Library/Formula/subversion.rb

```diff
@@ -70,6 +70,18 @@ class Subversion < Formula
   end
 
   def install
+    # OS X's Python is built universally and can't link with Homebrew's deps
+    # unless Homebrew's deps are universal as well.
+    # https://github.com/Homebrew/homebrew-versions/issues/777
+    if build.with?("python") && !File.exist?(HOMEBREW_PREFIX/"bin/python")
+      unless build.universal?
```

Comment:
I was mimicking [this block](https://github.com/DomT4/homebrew/blob/svn/Library/Formula/subversion.rb#L121-L133) further down the formula, but I'm happy to change if you're cool with it?